src/arch/riscv/pcstate.hh

```diff
@@ -56,15 +57,28 @@ constexpr enums::RiscvType RV64 = enums::RV64;
 
 class PCState : public GenericISA::UPCState<4>
 {
-  private:
+  protected:
+    typedef GenericISA::UPCState<4> Base;
+
     bool _compressed = false;
     RiscvType _rvType = RV64;
```

Comment:
Should this be initialized with `isa->rvType()`?

Reply:
It has initialized in isa.newPCState https://github.com/gem5/gem5/blob/stable/src/arch/riscv/isa.hh#L90